framework/src/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.ts

```diff
@@ -57,6 +57,12 @@ export class SubmitMainchainCrossChainUpdateCommand extends BaseCrossChainUpdate
 			context.params,
 		);
 
+		if (params.certificate.length === 0 && isInboxUpdateEmpty(params.inboxUpdate)) {
+			throw new Error(
+				'A cross-chain update must contain a non-empty certificate and/or a non-empty inbox update.',
+			);
+		}
```

Comment:
> # An empty object has all properties set to their default values.

I think from the condition and the error it is obvious what the comment is saying, so it's better not to write redundant thing

> Also we can move this to verifyCommon, since it's repeating in SidechainCrossChainUpdate Verification
https://github.com/LiskHQ/lips/blob/main/proposals/lip-0053.md#verification-1

this is kept at the top because that's the first check which can be done quickly and to mitigate the possibility on the following logics. If we move that to the `verifyCommon` the following logic must consider the case both are empty